filterList.php

```diff
@@ -47,28 +76,51 @@
         foreach($filtered_tasks as $task){
             echo "<br>Title:".$task['taskTitle']."<br>";
             echo "Description: ".$task['description']."<br>";
-    
+            echo "Importance: ".$task['importance']."<br>";
+            echo "Type of Work: ".$task['typeOfWork']."<br>";
+            echo "Due Date: ".gmdate("Y-m-d", $task['dueDate'])."<br>";
         }
     } //dealing with no result outputs(edge cases tested)
     else {
     echo "<br> No results found - please try again. <br>";
     }
-
+ 
 }
 ?>
 
 <!-- Form to take in user inputs -->
 <form action="filterList.php?id=<?php echo $_GET['id']; ?>" method="POST"> 
 
     <br>
-    <input type="text" name="filt_name" value="" class="form-control" placeholder="Task Title"/>
+    <input type="text" name="filt_name" value="" class="form-control" placeholder="Task Title" maxlength="255"/>
     <br>
-    <input type="number" name="filt_effort" value="" class="form-control" placeholder="Importance"/>
+    <input type="number" name="filt_effort" value="" class="form-control" placeholder="Importance" min = "0" max = "10" step="1"/>
     <br>
-    <input type="text" name="filt_work" value="" class="form-control" placeholder="Type of Work"/>
+    <input type="text" name="filt_work" value="" class="form-control" placeholder="Type of Work" maxlength="255"/>
     <br>
     <input type="datetime-local" name="filt_date" value="unchanged" class="form-control" placeholder="Due Date"/>
     <br>
+    <select name="sortTask">
+        <option value="" disabled selected>Sort Task By</option>
+        <optgroup label="Sort by Title">
+            <option value="ascTitle">Ascending</option>
+            <option value="descTitle">Descending</option>
+        </optgroup>
+        <optgroup label="Sort by important">
+            <option value="ascEffort">Ascending</option>
+            <option value="descEffort">Descending</option>
+        </optgroup>
+        <optgroup label="Sort by work type">
+            <option value="ascType">Ascending</option>
+            <option value="descType">Descending</option>
+        </optgroup>
+        <optgroup label="Sort by date">
```

Comment:
This should be 'Sort by due-date'

Reply:
Will make changes, thanks for the catch!